crates/ty_python_semantic/resources/mdtest/implicit_type_aliases.md

```diff
@@ -191,13 +191,13 @@ def _(
     reveal_type(int_or_callable)  # revealed: int | ((str, /) -> bytes)
     reveal_type(callable_or_int)  # revealed: ((str, /) -> bytes) | int
     # TODO should be Unknown | int
-    reveal_type(type_var_or_int)  # revealed: T@_ | int
+    reveal_type(type_var_or_int)  # revealed: typing.TypeVar | int
```

Comment:
We still don't support generic implicit type aliases, but this is a step in the right direction. `T@_` is just wrong.